Client/Frontend/Intro/IntroViewModel.swift

```diff
@@ -45,7 +45,11 @@ struct IntroViewModel: OnboardingViewModelProtocol {
     }
 
     var enabledCards: [InformationCards] {
-        return [.welcome, .wallpapers, .signSync]
+        guard let wallpaperVersion: WallpaperVersion = featureFlags.getCustomState(for: .wallpaperVersion) else {
```

Comment:
I will have to update to use a another feature flag to hide the card but I will keep the logic